app/assets/stylesheets/shared/newsletter-subscribe.scss

```diff
@@ -0,0 +1,57 @@
+.newsletter-form {
+  display: flex;
+  width: 100%;
+
+  input {
+    border-bottom-left-radius: $form-border-radius;
+    border-bottom-right-radius: 0;
+    border-top-left-radius: $form-border-radius;
+    border-top-right-radius: 0;
+
+    flex-grow: 1;
+    height: 40px;
+    vertical-align: top;
+  }
+
+  button {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: $base-border-radius;
+    border-top-left-radius: 0;
+    border-top-right-radius: $base-border-radius;
+
+    display: table-cell;
+    font-size: 1em;
+    height: 40px;
+    line-height: 1.3;
+    margin: 0;
+    padding: 0.25em 1em;
+    vertical-align: top;
+
+    i.fa {
+      color: white;
+      margin: 0;
+    }
+  }
+}
+
+.loading {
+  margin: 10% auto;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered border-bottom, border-left, border-radius, border-right, border-top, height, margin, width